src/styles/getModalStyles.js

```diff
@@ -100,7 +100,7 @@ export default (type, windowDimensions, popoverAnchorPosition = {}) => {
                 borderTopLeftRadius: 20,
                 borderTopRightRadius: 20,
                 paddingTop: 12,
-                paddingBottom: 12,
+                paddingBottom: 0,
```

Comment:
Haven't tested locally, but are you sure this doesn't cause regressions for the other BottomDocked modals, such as the `CreateMenu`? Also, I think you could maybe just remove this instead of explicitly giving it a value of 0.